workshops/guided-projects/guided_project_2.ipynb

```diff
@@ -0,0 +1,831 @@
+{
```

Comment:
<p>No change recommended, just wanted to call out I really like the flow you created in these sequential exercises of having learners make a change, re-run the unit tests to see the failure, and understand the failure before proceeding with fixing. Nice job!</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/mlops-on-gcp/pull/31/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='46'/>

Reply:
<p>Glad you like it!</p>